cvat/apps/dataset_manager/task.py

```diff
@@ -35,6 +35,7 @@
 from cvat.apps.profiler import silk_profile
 
 dlogger = DatasetLogManager()
+ANNOTATIONS_CHUNK_SIZE = 2000
```

Comment:
Typically it makes sense to make such performance constants easily configurable. Consider putting it into `cvat/apps/engine/default_settings.py` and using `os.getenv` to allow passing via an env variable.